mappings/limes_v2.38.yaml

```
@@ -0,0 +1,121 @@
+model: LIMES 2.38
+native_regions:
+ - AUT: Austria
+ - BEL: Belgium
+ - BGR: Bulgaria
+ - CHE: Switzerland
+ - CZE: Czech Republic
+ - DEU: Germany
+ - DNK: Denmark
+ - ESP: Spain
+ - EST: Estonia
+ - FIN: Finland
+ - FRA: France
+ - GBR: United Kingdom
+ - GRC: Greece
+ - HRV: Croatia
+ - HUN: Hungary
+ - IRL: Ireland
+ - ITA: Italy
+ - LTU: Lithuania
+ - LUX: Luxembourg
+ - LVA: Latvia
+ - NLD: The Netherlands
+ - NOR: Norway
+ - POL: Poland
+ - PRT: Portugal
+ - ROU: Romania
+ - SVK: Slovakia
+ - SVN: Slovenia
+ - SWE: Sweden
+common_regions:
+  - EU27:
+    - Austria

```

This has to use the region names as reported by the model (i.e., uploaded to the IIASA database)
```suggestion
    - AUT
```